lib/system_win32.c

```diff
@@ -89,6 +95,16 @@ CURLcode Curl_win32_init(long flags)
   }
 #endif
 
+  s_hIpHlpApiDll = Curl_load_library(TEXT("iphlpapi.dll"));
+  if(s_hIpHlpApiDll) {
+    /* Get the address of the if_nametoindex function */
+    IF_NAMETOINDEX_FN pIfNameToIndex = CURLX_FUNCTION_CAST(IF_NAMETOINDEX_FN,
+                                          (GetProcAddress(s_hIpHlpApiDll,
+                                                          "if_nametoindex")));
```

Comment:
This looks a bit funny indented. I presume because it got so wide. In these case I would say
breaking the line after `=` is the saner approach so that the `CURLX_` can start indented just two columns in on the following line.